test-node.bash

```diff
@@ -365,7 +377,15 @@ if $force_init; then
         echo == Funding l3 funnel and dev key
         docker-compose up -d l3node poster
 
-        if ! $l3CustomFeeToken; then
+        if $l3_token_bridge; then
+            echo == Deploying L2-L3 token bridge
+            rollupAddress=`docker-compose run --entrypoint sh poster -c "jq -r '.[0].rollup.rollup' /config/deployed_l3_chain_info.json | tail -n 1 | tr -d '\r\n'"`
+            docker-compose run -e ROLLUP_ADDRESS=$rollupAddress -e PARENT_RPC=http://sequencer:8547 -e CHILD_RPC=http://l3node:3347 tokenbridge deploy:local:token-bridge
```

Comment:
what would be the deployer of this script?

Reply:
Script by default uses deployer derived from 'user_token_bridge_deployer'

Reply:
ah right https://github.com/OffchainLabs/token-bridge-contracts/pull/50/files
can we use an argument (or better, env) to pass the private key instead?

Reply:
or maybe just pass the account name "user_token_bridge_deployer"
instead of hardcoding in the local deploy script

Reply:
Added passing the private key.  We could pass account names, but that would involve different parsing logic in deployment script for different inputs (ie. in case of deploying L1-L2 bridge we don't use account name, but hard-coded key)

Reply:
So if we keep this approach I will remove defaults from deployer script

Reply:
LGTM